extensions/amp-jwplayer/0.1/amp-jwplayer.js

```diff
@@ -82,12 +82,10 @@ class AmpJWPlayer extends AMP.BaseElement {
 
     this.contentSearch_ = element.getAttribute('data-content-search') ||
         '';
-    this.contentContextual_ = element.getAttribute('data-content-contextual') ||
-        false;
+    this.contentContextual_ = element.hasAttribute('data-content-contextual');
```

Comment:
this would behave differently for `data-content-contextual=false` (since the value gets passed as query string param) which per documentation is valid. I suggest changing types to string and using `''` as default.

Reply:
Done.